### ISSUE TITLE
bokehservice container setup

### ISSUE THREAD

**Issue by caseyjlaw** [dsa110/dsa110-issues#31]
After the recent restart of containers, I see bokehservice needs some more work to run bokeh antmon tool.

1. The setup for bokehservice needs to run
`conda install bokeh`

2. The new syslog version (James's new syntax) isn't tagged right. It is in development only. Is the solution to tag and merge it to main?

**Comment by caseyjlaw** [comment]
@rh-codebase The bokehservice conatainer is broken again (after restart?). The annotated tagging of dsa110-pyutils isn't set up right, so the development branch isn't set up as expected. Can you tag and merge that change for the new API?

**Comment by rh-codebase** [comment]
are you referring to the repo in  bokehservice or github? In github I still don't see a version tag on your merge between origin/developmment and origin/cjl/dev. If you see your version tag locally, try pushing with --tags option.

**Comment by caseyjlaw** [comment]
Ok, I added tag v2.7.0 to master, which is the same as development now. Take a look and let me know what you think.

**Comment by rh-codebase** [comment]
Yikes. somewhere along the development path, the tests broke(or my environment). README.md states to:
cd test
 coverage run -m pytest
When I tried this at v2.7.0, 4 tests failed.  
Maybe my environment???

**Comment by rh-codebase** [comment]
Yep. my environment. I'll update the README to state to be sure to pip install . at TOT before running test. All tests pass. at v2.7.0 for me. Doesn't look like the Documentation has been updated(running sphinx) and commit.

**Comment by rh-codebase** [comment]
bumped version: v2.7.1 includes updated Docs and test results.

**Comment by rh-codebase** [comment]
Got sidetracked with pyutils. The original issue was the need to run:
conda install bokeh

which still needs to be added to the bokeh container profile.

**Comment by caseyjlaw** [comment]
I actually added a reply to another ticket about that, I think. But yeah, we can close two of we can get a chance to define special classes of container.

**Comment by rh-codebase** [comment]
Ah. Nice. We can close this then with a ref.to that issue. There must be a way to add a linked ref to another issue.

**Comment by caseyjlaw** [comment]
Special containers will be dealt with in issue #32

**Comment by caseyjlaw** [comment]
Sorry, but I need to reopen this issue. I found that bokeh wasn't running in the container after a recent restart.
The bokehservice container needs one more thing to restart automatically. Currently, I need to do this:
`nohup bokeh serve antmon.py &`

But could that be done automatically? Or would you prefer to use serviced?
